ee/api/session_recording_playlist.py

```diff
@@ -223,11 +224,16 @@ def modify_recordings(
 
         # TODO: Maybe we need to save the created_at date here properly to help with filtering
         if request.method == "POST":
-            recording, _ = SessionRecording.objects.get_or_create(session_id=session_recording_id, team=self.team)
+            recording, _ = SessionRecording.objects.get_or_create(
+                session_id=session_recording_id, team=self.team, deleted=False
```

Comment:
@alexkim205 FYI - we should not non-indexable properties this way. `deleted=False` should be set in the "defaults" property. 
```suggestion
                session_id=session_recording_id, team=self.team, defaults={'deleted'=False}
```

https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create